assets/data/cards_pt-br.json

```
@@ -296,6 +296,21 @@
       "title": "Pull Request",
       "description": "Trata-se de uma requisição de alteração dentro de um projeto. Essa requisição pode ser feita para corrigir um problema, adicionar uma nova funcionalidade, sugerir alterações, entre outros motivos. Essa requisição de alteração é feita para que haja uma avaliação do que está sendo alterado dentro do projeto, entendendo de faz sentido para o contexto daquele momento. E essa requisição pode ficar pendente até que pessoas responsáveis ou que tenham a permissão, aprovem. Outras regras podem ser adicionadas nessas requisições, como build da aplicação, rodar testes com sucesso e o que mais fizer sentido para o projeto em questão.",
       "tags": ["Conceito"]
-    }    
+    },
+    {
+      "title": "Garbage Collector",
+      "description": "Literalmente \"coletor de lixo\" em inglês, é um tipo de software auxiliar que é executado ao lado de uma aplicação principal, percebendo memória que foi alocada (designada ao programa) mas não é mais necessária, permitindo que seja retornada ao sistema operacional. Geralmente, é usado por determinação conta da linguagem utilizada, como Java, C# e Javascript",
+      "tags": ["Conceito"]
+    },
+    {
+      "title": "Computação em Nuvem",
+      "description": "De modo geral, refere-se ao uso de computação disponível através da internet e abstraído do equipamento físico utilizado. É um conceito extremamente amplo, que compreende desde armazenamento pessoal até aplicações que são executadas na nuvem. O conceito de computação em nuvem contrasta com a ideia de computação on-premise, que se dá quando o usuário também administra o equipamento que disponibiliza um serviço.",
+      "tags": ["Conceito"]
+    },
+    {
+      "title": "CORS",
+      "description": "Do inglês \"Cross-Origin Resource Sharing\", ou Compartilhamento de Recursos de Origem Cruzada, refere-se a um mecanismo que gere recursos web acessados a partir de outras páginas. Recursos como imagens e arquivos CSS são permitidos por padrão pelo CORS, mas scripts são bloqueados, o que frequentemente atrapalha desenvolvedores iniciantes.",
```

Gostaria de sugerir uma versão um pouco mais simplificada, que poderia ajudar especialmente as pessoas que estão começando a entender melhor este conceito. Segue minha sugestão:

```suggestion
      "description": "Do inglês \"Cross-Origin Resource Sharing\" ou, em português, \"Compartilhamento de Recursos de Origem Cruzada\", é uma regra de segurança para websites. Ele determina como recursos de uma página web podem ser acessados por outra página de um domínio diferente. Por exemplo, uma imagem ou arquivo CSS geralmente pode ser compartilhado entre diferentes sites sem problemas, mas códigos JavaScript frequentemente têm restrições mais rigorosas.",
```